README.md

```diff
@@ -52,11 +54,13 @@ Install or make sure Chrome browser (at least) have it in your development machi
 |---- reports (containing html report & assets)
 |---- logs(contains logs of the test run)
 |---- node_modules
+|---- scripts
+|   |____ script.sh (scripts used in package.json)
+|   |____ set-env-vars.sh (set environment variables in CI)
 |---- .editorconfig
 |---- .gitattributes
 |---- .gitignore
 |---- reporter.js (for generating html reports)
-|---- cypress.env.template.json (credentials & environment variables - rename the file as mentioned above)
```

Comment:
In the project folder there is still cypress.env.template.json exist, should it be removed from there too?

Reply:
read again complete README in this PR.

Reply:
which line number in README you meant to read again that answer my question, can you please describe bit more?

Reply:
I no longer see use of this file in the project folder? where/when does this file required? -> cypress.env.template.json